src/DIRAC/TransformationSystem/Utilities/ReplicationTransformation.py

```diff
@@ -1,8 +1,46 @@
 """
 Utilities to create replication transformations
 """
-from DIRAC.TransformationSystem.Client.Transformation import Transformation
 from DIRAC import gLogger, S_OK, S_ERROR
+from DIRAC.TransformationSystem.Client.Transformation import Transformation
+from DIRAC.TransformationSystem.Client.TransformationClient import TransformationClient
+
+
+def getTransformationName(transName, unique=True):
+    """
+    Get a transformation name from a base name
```

Comment:
```suggestion
    Get a transformation name from a base name.
    
```
add empty line after short summary of function.